layouts/shortcodes/observability_pipelines/processors/generate_metrics.md

```diff
@@ -1 +1,17 @@
-REUSE INSTRUCTIONS
\ No newline at end of file
+This processor generates either a count metric of logs that match a query or a distribution metric of a numeric value contained in the logs, such as a request duration.
+
+To set up the processor:
+
+Click **Manage Metrics** to create new metrics or edit existing metrics. This opens a side panel where:
+
+- If you have not created any metrics yet, enter the metric parameters as described in the [Add a metric](#add-a-metric) section to create a metric.
+- If you have already created metrics, click on the metric's row in the overview table to edit or delete it. You can search for a specific metric by its name, using the search bar, and then select the metric to edit or delete it. Click **Add Metric** to add another metric.
+
+##### Add a metric
+
+ 1. Enter a [filter query](#filter-query-syntax). Only logs that match the specified filter query are processed. All logs, regardless of whether they do or do not match the filter query, are sent to the next step in the pipeline. **Note**: Since a single processor can generate multiple metrics, you can define a different filter query for each metric.
```

Comment:
```suggestion
 1. Enter a [filter query](#filter-query-syntax). Only logs that match the specified filter query are processed. All logs, regardless of whether they match the filter query, are sent to the next step in the pipeline. **Note**: Since a single processor can generate multiple metrics, you can define a different filter query for each metric.
```